paypal/paypal.lua

```diff
@@ -0,0 +1,101 @@
+--[[
+
+# [PayPal API](https://developer.paypal.com/docs/api/overview/) connector.
+
+### Authorization
+
+Authorization use [OAuth 2.0](https://developer.paypal.com/docs/api/overview/#authentication-and-authorization).
+
+## Input parameters
+
+* `client_id` - paypal client ID
+
+* `secret` - paypal secret.
+* `grant_type` - client_credentials.
+
+* api_endpoint - API endpoint URI, e.g. `payment`.
+* method - optional request method, defauls to `GET`, could be `GET`, `POST`, `PUT`, `DELETE`.
+* post_data - optional JSON string used to post data.
+
+## Output parameters
+
+All connector output parameters could be used in PMIO Service Task output parameters as a value placeholders in curly braces.
+
+Example connector output parameters:
+
+```
+{
+  "payments":
+    {
+      "id": "PAY-0US81985GW1191216KOY7OXA",
+      "create_time": "2017-06-30T23:48:44Z",
+      "update_time": "2017-06-30T23:49:27Z",
+      "state": "approved",
+      "intent": "order",
+      "payer": {
+        "payment_method": "paypal"
+    }
+}
+```
+
+* response - JSON decoded structure received from API response.
+* code - HTTP response code.
+* headers - headers structure received from API response.
+
+]]--
+
+local https = require("ssl.https")
+local ltn12 = require"ltn12"
+local cjson = require("cjson")
+
+local mime = require("mime")
+
+local inputVar = cjson.decode(io.stdin:read("*a"))
+
+local respbody = {}
+local reqbody = inputVar.post_data and cjson.encode(inputVar.post_data) or ''
+
+local paypalDomain = inputVar.paypal_domain
+local clientId = inputVar.client_id
+local clientSecret = inputVar.client_secret
+
+r, c, h = https.request{
+    method = 'POST',
+    url = paypalDomein .. '/v1/oauth2/token',
```

Comment:
now its broken, you defined variable as paypalDomain, but left here the old variable name with misspell.